packages/messenger-widget/src/hooks/haltDelivery/useHaltDelivery.ts

```diff
@@ -145,8 +155,13 @@ export const useHaltDelivery = () => {
 
             await submitEnvelopsToReceiversDs(dispatchableEnvelops);
 
+            // delete the halted messages with messageId
             dispatchableEnvelops.map((envelop) => {
-                clearHaltedMessages(envelop.haltedEnvelopId, envelop.aliasName);
+                // fetch the messageId from message signature
+                if (envelop.envelop.metadata?.signature) {
+                    const msgId = msgMap.get(envelop.envelop.message.signature);
```

Comment:
Isn't haltedEnevelpeId defined in 138 the same as msgId in 162?

Reply:
No, those are different